Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -59,6 +63,10 @@ public class CustomerIOImplementation: CustomerIOInstance {
         handler(&configToModify)
 
         sdkConfigStore.config = configToModify
+
+        if sdkConfigStore.config.autoTrackScreenViews {
+            enableAutoScreenviewTracking()
+        }
```

Comment:
Because this code exists, I don't see a value in adding `enableAutoScreenviewTracking()` to `CustomerIOInstance` protocol that is public facing that a customer could use. I find it confusing to have multiple places where a customer could configure the SDK for automatic screen view tracking.